storage.lua

```diff
@@ -246,6 +247,57 @@ function mail.extractMaillists(receivers_string, maillists_owner)
 	return receivers
 end
 
+function mail.get_setting_default_value(setting_name)
+	local default_values = {
+		chatnotif = true,
+		onjoinnotif = true,
+		hudnotif = true,
```

Comment:
Personally in this case I would rather go with long names than semi abbreviations.
I mean this (but it would of course require changing these elsewhere too):
```suggestion
		chat_notifications = true,
		onjoin_notifications = true,
		hud_notifications = true,
```
I can however live with this too, so not demanding changes unless you've been thinking same thing.

If you'd like to change anyway, something like this this should work (in mail mod dir): `find . -type f -name '*.lua' -print0 | xargs -0 sed -ri 's/([a-z]+)notif([^a-z])/\1_notifications\2/g'`

Reply:
I followed your advice, this is a good idea, and the command line worked well, thx =)